flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_ATC_AtisMenu.js

```diff
@@ -0,0 +1,228 @@
+class CDUAtcAtisMenu {
+    static CreateDataBlock(mcdu) {
+        const airports = [
+            { icao: "", type: Atsu.AtisType.Departure, requested: false, autoupdate: false },
+            { icao: "", type: Atsu.AtisType.Arrival, requested: false, autoupdate: false },
+            { icao: "", type: Atsu.AtisType.Arrival, requested: false, autoupdate: false },
+            { icao: "", type: Atsu.AtisType.Arrival, requested: false, autoupdate: false }
+        ];
+
+        if (mcdu.flightPlanManager.getOrigin() && mcdu.flightPlanManager.getOrigin().ident) {
```

Comment:
Should this use getPersistentOrigin? I think it was added because getOrigin goes away after DIR TO or something.

Reply:
fixed